man/systemctl.xml

```diff
@@ -1652,6 +1652,9 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
             <para>This command honors <option>--force</option> and <option>--when=</option> in a similar way
             as <command>halt</command>.</para>
 
+            <para>If a new kernel has been loaded via <command>kexec --load</command>, a
```

Comment:
So this is wrong. First, you've only changed the behavior of sysv-compat commands, i.e. direct calls to `shutdown`/`reboot`/`halt`. Secondly, is this really the right thing to do? IIUC this is compat break for logind.